src/main/java/me/zeroeightsix/kami/module/modules/render/MapPeview.kt

```diff
@@ -0,0 +1,41 @@
+package me.zeroeightsix.kami.module.modules.render
+
+import me.zero.alpine.listener.EventHandler
+import me.zero.alpine.listener.EventHook
+import me.zero.alpine.listener.Listener
+import me.zeroeightsix.kami.module.Module
+import me.zeroeightsix.kami.setting.Settings
+import net.minecraft.client.renderer.GlStateManager
+import net.minecraft.item.ItemMap
+import net.minecraft.world.World
+import net.minecraftforge.client.event.RenderTooltipEvent
+
+
+/**
+ * Created by fred41 on 27/05/2020.
+ *
+ */
+@Module.Info(
+        name = "MapPreview",
+        category = Module.Category.RENDER,
+        description = "Shows the contents of a map when you hover over one"
+)
+class MapPeview : Module() {
+    private val scale = register(Settings.doubleBuilder("scale").withRange(0.0, 3.0).withValue(0.5).build())
+
+    @EventHandler
+    private val tooltipEventListener = Listener(EventHook { event: RenderTooltipEvent.PostText ->
+        if (event.getStack().getItem() is ItemMap) {
+            val mapdata = (event.getStack().getItem() as ItemMap).getMapData(event.getStack(), mc.world as World)
+            if(mapdata==null)
+                return@EventHook
+            GlStateManager.disableDepth()
```

Comment:
a bit nit picky, but change it to this please
```kotlin
if (mapdata == null) return@EventHook
```
and leave a space before the GL stuff

Reply:
i can run the default intellij reformat code  is that ok ?

Reply:
sure.